### PR TITLE
feat(collect-data-modal): apply duplication case and component separation

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/CollectorDataModal.vue [hunk 1]
```diff
@@ -76,12 +139,41 @@ const collectorPageState = collectorPageStore.$state;
...
+    definitionData: computed(() => ({
+        account: '123',
+        duration: 10,
+        status: {
+            succeededCount: 5,
+            failedCount: 2,
+            totalCount: 15,
+            succeededPercentage: 50,
+            failedPercentage: 20,
+        },
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/CollectorDataModal.vue]: nitpick: Please comment that it is sample data!

**Review by WANZARGEN — APPROVED** [review]
It looks good overally!

**Review by WANZARGEN — COMMENTED** [review]
Oh but it seems an issue!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Review by WANZARGEN — COMMENTED** [review]
Oh but it seems an issue!

**Pull request merged**